### PR TITLE
Places view

### PR DESCRIPTION
**Pull request by TomaszKotowski**

### CODE REVIEW

File: src/models/places/places.view.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+  @bind
+  handlePlacesFilter = (optionValue) => {
+    this.filterOfficeId = optionValue;
+    this.office = this.props.officesStore.officesList.find((office) => office.id === optionValue);
+  }
+
+  //  remove later
+  componentDidMount() {
+    console.log("componentDidMount");
```
Comment (kamil7x) [inline src/models/places/places.view.js]: Remove unnecessary `console.log`s

File: src/stores/OfficesStore.js [hunk 2]
```diff
@@ -43,7 +46,7 @@ class OfficesStore {
    */
   findOfficeByCityName(cityName) {
     const office = find(this.officesList, element => {
-      element.name === cityName 
+      element.name === cityName
```
Comment (kamil7x) [inline src/stores/OfficesStore.js:47]: You are not returning anything

File: src/stores/PlacesStore.js [hunk 3]
```diff
@@ -28,15 +31,15 @@ class PlacesStore {
    * Find place by it's id. Return place object or empty string
    * @param {string} placeId 
    */
-  findById(placeId){
+  findById(placeId) {
     const result = find(this.placesList, el => {
       el.id === placeId
```
Comment (kamil7x) [inline src/stores/PlacesStore.js]: Is it working? I think I missed this last time.

**Review by kamil7x — CHANGES_REQUESTED** [review]

**Comment by TomaszKotowski** [comment]
Fixed the files and remove unnecessary console.log

**Review by kamil7x — APPROVED** [review]

**Pull request merged**